SymbolCollisionTest/Podfile

```diff
@@ -25,9 +25,6 @@ target 'SymbolCollisionTest' do
     pod 'FirebaseInstanceID', :path => '../'
     pod 'FirebaseInstallations', :path => '../'
     pod 'FirebaseMessaging', :path => '../'
-    pod 'Firebase/MLCommon', :path => '../'
-    pod 'Firebase/MLModelInterpreter', :path => '../'
-    pod 'Firebase/MLVision', :path => '../'
```

Comment:
Originally thought that we should keep these but realized with Firebase versioning it'll be impossible to get a configuration with these pods and Firebase 8+, so I'm good with removing them.